index.html

```diff
@@ -7935,7 +7935,8 @@ <h2 id="MediaStreamAudioSourceNode">
           connections to the input are up-mixed/down-mixed to the number of
           channels of the input.
         </p>
-        <dl title="interface MediaStreamAudioDestinationNode : AudioNode"
+        <dl title=
+        "[Constructor(AudioContext context, optional AudioNodeChannelOptions options)]interface MediaStreamAudioDestinationNode : AudioNode"
```

Comment:
I've never used a MediaStreamAudioDestinationNode, but from reading the
spec, it seems that the stream attribute is created when the node is
created; it's not a input or anything..

On Sun, Jul 31, 2016 at 9:37 AM, Joe Berkovitz <notifications@github.com>
wrote:

> In index.html
> <https://github.com/WebAudio/web-audio-api/pull/898#discussion_r72906956>:
>
> > @@ -7935,7 +7935,8 @@ <h2 id="MediaStreamAudioSourceNode">
> >            connections to the input are up-mixed/down-mixed to the number of
> >            channels of the input.
> >          </p>
> > -        <dl title="interface MediaStreamAudioDestinationNode : AudioNode"
> > +        <dl title=
> > +        "[Constructor(AudioContext context, optional AudioNodeChannelOptions options)]interface MediaStreamAudioDestinationNode : AudioNode"
>
> This appears to be lacking a property bag in which to provide the initial
> MediaStream destination, following the handling of media stream sources
> in #897 <https://github.com/WebAudio/web-audio-api/pull/897> .
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/WebAudio/web-audio-api/pull/898/files/feb33228ec9c4d02f901c17036c4869530dcbe04#r72906956>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAofPCZsvEE2hACzPGZO0VdRrn26Sw4Aks5qbM8-gaJpZM4JHYOa>
> .
>



-- 
Ray